content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -15,12 +15,44 @@ menu:
 
 # Upgrade from Gogs
 
-If you are running Gogs which version is below 0.9.99, you can upgrade Gogs to Gitea smoothly. There are some steps to do below:
+If you are running Gogs which version is below 0.9.99, you can upgrade Gogs to Gitea smoothly. There are some steps to do below. Make sure to complete as the gitea user in ~:
 
 * Stop Gogs
-* Copy Gogs `custom/conf/app.ini` to Gitea.
-* Copy Gogs `conf/` to `options/` for Gitea.
-* If you have more custom content on `custom/` folder, like templates or localization files, you should carrefully merge manually because these files changed on Gitea.
-* Copy `data/` folder from Gogs to Gitea. It contains issue attachments and avatars.
-* Run Gitea
-* Enter Gitea admin panel on the UI, run `Rewrite '.ssh/authorized_keys' file` (caution: non-Gitea keys will be lost) and `Rewrite all update hook of repositories` (needed when custom config path is changed).
\ No newline at end of file
+* cd ~
+* wget https://github.com/go-gitea/gitea/archive/v1.0.0.zip
+* unzip v1.0.0.zip
+* cp -r gogs gitea
+* If you have custom content migrated from gogs/custom/ folder, like templates or localization files, you should manually merge these because these files changed on Gitea.
```

Comment:
The custom folder still applies for Gitea, just move `gogs/custom/conf` (not `gogs/custom/conf/app.ini`) to `gitea/custom/options`, this is used to override gitignores, labels, licenses, locales or readmes.